homeassistant/components/london_underground/coordinator.py

```diff
@@ -24,7 +28,7 @@ def __init__(self, hass, data):
         )
         self._data = data
 
-    async def _async_update_data(self):
+    async def _async_update_data(self) -> Any:
```

Comment:
Would it be possible to type more specifically (eg at least 'dict')?

The library is not strictly typed but appears to be `Dict[str, Dict[str, str]]`